website/content/v1.10/introduction/getting-started.md

```diff
@@ -1,323 +1,229 @@
 
-For example, the `talosctl get disks` command below shows that the system has a `vda` drive, not an `sda`:
+## Step 5: Learn About Your Installation Disks 
 
-```sh
-$ talosctl -n 192.168.0.2 get disks --insecure
-DEV        MODEL   SERIAL   TYPE   UUID   WWID  MODALIAS                    NAME   SIZE    BUS_PATH
-/dev/vda   -       -        HDD    -      -      virtio:d00000002v00001AF4   -      69 GB   /pci0000:00/0000:00:06.0/virtio2/
-```
+When you first boot your machine from the ISO, Talos runs temporarily in memory. This means that your Talos nodes, configurations, and cluster membership won't survive reboots or power cycles.
 
-In this case, you would modify the `controlplane.yaml` and `worker.yaml` files and edit the line:
+However, once you apply the machine configuration (which you'll do later in this guide), you'll install Talos, its complete operating system, and your configuration to a specified disk for permanent storage.
 
-```yaml
-install:
-  disk: /dev/sda # The disk used for installations.
+Run this command to view all the available disks on your control plane:
 ```
+talosctl get disks --insecure --nodes $CONTROL_PLANE_IP
+```
+Note the disk ID (e.g., `sda`, `vda`) as you will use it in the next step.
 
-to reflect `vda` instead of `sda`.
-
-> For information on customizing your machine configurations (such as to specify the version of Kubernetes), using [machine configuration patches]({{< relref "../talos-guides/configuration/patching" >}}), or customizing configurations for individual machines (such as setting static IP addresses), see the [Production Notes]({{< relref "prodnotes#customizing-machine-configuration" >}}).
-
-## Accessing the Talos API
-
-Administrative tasks are performed by calling the Talos API (usually with `talosctl`) on Talos Linux control plane nodes, who may forward the requests to other nodes.
-Thus:
 
-- ensure your control plane node is directly reachable on TCP port 50000 from the workstation where you run the `talosctl` client.
-- until a node is a member of the cluster, it does not have the PKI infrastructure set up, and so will not accept API requests that are proxied through a control plane node.
 
-Thus you will need direct access to the **worker** nodes on port 50000 from the workstation where you run `talosctl`  in order to apply the initial configuration.
-Once the cluster is established, you will no longer need port 50000 access to the workers.
-(You can avoid requiring such access by passing in the initial configuration in one of other methods, such as by cloud `userdata` or via `talos.config=` kernel argument on a `metal` platform)
+## Step 6: Generate Cluster Configuration
 
-This may require changing firewall rules or cloud provider access-lists.
+Talos Linux is configured entirely using declarative configuration files rather than traditional SSH commands.
```

Comment:
```suggestion
Talos Linux is configured entirely using declarative configuration files avoiding the need to deal with SSH and running commands.
```

"SSH commands" doesn't make sense as ssh is a protocol used to run the commands on a remote machine.